pages/cadre.tsx

```diff
@@ -31,7 +29,9 @@ const TheCadrePage = () => {
   const [loading, setLoading] = useState(true)
   const [members, setMembers] = useState<TeamMember[]>([])
 
+
   useEffect(() => {
+    setLoading(true);
```

Comment:
It looks like state is still being used on this page rather than the context.

Reply:
thank you. It's updated.